src/shared.js

```diff
@@ -3,3 +3,18 @@ import { css } from "styled-components";
 export const Outline = css`
   box-shadow: 0 3px 5px rgba(0, 0, 255, 0.3);
 `;
+
+export const FONT_WEIGHT = {
+  light: "300",
+  normal: "400",
+  medium: "500",
+};
+
+export const PALETTE = {
```

Comment:
The confirmed colors we have so far from Docs

Reply:
Since this'll be a public API, this might be a good opportunity to start using a naming scheme like `PALETTE.black[-1]` etc. -4 to 4? Can fill in exact colors later, but it'd be good to have a pattern with slots to fill in.

Reply:
The Palette color names are based on how designer named it on Zeplin. On Zeplin, as color shades get darker, its integer increases unless it's complete black or white. Lighter black is `black60`, darker black is `black80`